pkg/agent/flowexporter/connections/conntrack_connections.go

```diff
@@ -174,7 +177,10 @@ func (cs *ConntrackConnectionStore) Poll() ([]int, error) {
 	for _, conn := range filteredConnsList {
 		cs.AddOrUpdateConn(conn)
 	}
-	cs.fillL7EventInfo(l7EventMap)
+	if l7EventMap != nil && len(l7EventMap) != 0 {
```

Comment:
`if len(l7EventMap) != 0 {` is sufficient
for a nil map, len will return 0

Reply:
Fixed. Thanks.